README.md

```diff
@@ -295,6 +295,201 @@ To graceful shutdown a Presto cluster without query losses, the steps are:
 
 To gracefully shutdown a single worker process, see [this](https://trino.io/docs/current/admin/graceful-shutdown.html) for the operations.
 
+## Routing Rules Engine
+By default, presto-gateway reads the `X-Trino-Routing-Group` request header to route requests.
+If this header is not specified, requests are sent to adhoc.
```

Comment:
Change this "requests are sent to adhoc." to 
"requests are sent to default routing group(adhoc)"

Reply:
done!